ddtrace/tracer/option_test.go

```diff
@@ -433,19 +434,22 @@ func TestTracerOptionsDefaults(t *testing.T) {
 		})
 		t.Run("env", func(t *testing.T) {
 			t.Setenv("DD_TRACE_DEBUG", "true")
+			internalconfig.ResetConfigForTesting()
```

Comment:
Have we looked into what the CI time might look like after adding this? I can look into this. If it's a lot, we can play around with the CI jobs we have to maybe separate `option_test.go` into its own job or something. This and the benchmark should help with adding more testing in the future 👍🏼